run/events/README.md

```diff
@@ -0,0 +1,35 @@
+# Events for Cloud Run – C# Quickstart
+
+This folder contains a quickstart app for using Events for Cloud Run with C#.
+
+## Deploy to Cloud Run
+
+```sh
+PROJECT_ID=$(gcloud config get-value project)
+MY_RUN_SERVICE=events-csharp
+gcloud builds submit --tag gcr.io/$PROJECT_ID/$MY_RUN_SERVICE
+gcloud run deploy $MY_RUN_SERVICE --image gcr.io/$PROJECT_ID/$MY_RUN_SERVICE --platform managed --allow-unauthenticated
+```
+
+Optionally test your Run service:
+
+```sh
+MY_RUN_SERVICE_URL=$(gcloud run services describe "$MY_RUN_SERVICE" --format 'value(status.address.url)')
+curl $MY_RUN_SERVICE_URL
+```
+
+Test with a Cloud Event:
+
+```sh
+curl $MY_RUN_SERVICE_URL \
```

Comment:
The examples I've seen typically use all lower-case for the CloudEvent headers (`ce-id`, `ce-type` etc).

Reply:
Ok, changed README to have lowercase CloudEvent headers